src/driver/driver.hpp

```diff
@@ -44,13 +44,13 @@ class Driver {
 class SimpleDriver : public Driver {
  public:
   SimpleDriver(ParameterInput *pin, Mesh *pm, Outputs *pout) : Driver(pin, pm, pout) {}
-  DriverStatus Execute() { return DriverStatus::complete; }
+  DriverStatus Execute() override { return DriverStatus::complete; }
```

Comment:
Why not declare as final?

Reply:
I wasn't sure what the intention was, so I was just trying to match the previous behavior, but more explicitly.